windows/whats_up.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
it would be great if you can have the same name "NewsReader" for windows version as well.